concepts/05 Widgets/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

```diff
@@ -0,0 +1,163 @@
+You can divide form items into several columns. To keep the fixed number of columns, specify the number in the [colCount](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#colCount) property. An item can occupy more than one column. For this, specify the [colSpan](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#colSpan) property. In the example below, the `Notes` item spans two columns:
```

Comment:
The folder "40 Organize the Form Items" requires an empty md with the same name inside. Otherwise, the link to this section in the left-hand tree will be broken.